interface/src/main.cpp

```diff
@@ -66,59 +66,219 @@ int main(int argc, const char* argv[]) {
 
     setupHifiApplication(BuildInfo::INTERFACE_NAME);
 
-    QStringList arguments;
-    for (int i = 0; i < argc; ++i) {
-        arguments << argv[i];
-    }
-
     QCommandLineParser parser;
     parser.setApplicationDescription("Vircadia");
```

Comment:
I know you didn't change it, but while we're at it, this could use being a bit more informative. As it is, we're getting:

```
Usage: interface/interface [options]
Vircadia

Options:
  -h, --help                               Displays help on commandline
                                           options.
```